scaladoc/test/dotty/tools/scaladoc/snippets/SnippetCompilerTest.scala

```diff
@@ -42,26 +43,22 @@ class SnippetCompilerTest {
   @Test
   def snippetCompilerTest: Unit = {
     val simpleCorrectSnippet = s"""
-      |package asd
       |class A:
       |  val b: String = "asd"
       |""".stripMargin
 
     val simpleIncorrectSnippet = s"""
-      |package asd
```

Comment:
Why we can't have packages here?

Reply:
because after wrapping we would get package definition in not-top level

Reply:
Do we have good mechanism to detect/report that? What if on has `package xxx` in his snippet?

Reply:
Snippet compiler will report error:
```
At 52:4:
  package abc
Error: illegal start of definition
```